haskell-backend/pom.xml

```diff
@@ -28,12 +28,42 @@
     </dependency>
   </dependencies>
 
+  <properties>
```

Comment:
I'm reasonably sure this is not necessary as I added this flag specifically to deal with cmake.